meta-lxatac-bsp/recipes-kernel/linux/files/defconfig

```diff
@@ -5710,13 +5736,7 @@ CONFIG_OVERLAY_FS=y
 #
 CONFIG_NETFS_SUPPORT=y
 CONFIG_NETFS_STATS=y
-CONFIG_FSCACHE=m
```

Comment:
I assume this is just part of olddefconfig?

Reply:
I guess so. At least it is nothing that I've actively touched.